app/src/main/java/com/github/se/cyrcle/ui/review/ReviewScreen.kt

```diff
@@ -49,6 +50,9 @@ fun ReviewScreen(
     reviewViewModel: ReviewViewModel,
     userViewModel: UserViewModel
 ) {
+  val configuration = LocalConfiguration.current
+  val screenWidth = configuration.screenWidthDp
```

Comment:
You could just do `screenWidth = configuration.screenWidthDp / 160f` because i don't think you use `screenWidth anywhere else.

Reply:
I feel that this makes the code slightly more readable!